FreeRTOS/FreeRTOS-Plus-TCP
# Add changes for Coverity fix

Fix Coverity Errors :

Test Steps
-----------
Run Coverity

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
